src/components/GameField/GameField.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useMemo, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { ChangeDirectionActions } from '../../store/types';
+import { directionActionCreator } from './../../store/actions';
+import './GameField.css'
+
+const GameField = () => {
+	const snakePosition = useSelector((state: {position: number[]}) => state.position);
+	const dispatch = useDispatch();
+	
+	const fieldCells = useMemo(() => {
+		const result = [];
+		for (let i = 1; i < 226; i++) {
+			result.push(i)
+		}
+		return result;
+	}, [])
+
+	const onKeyPressListener = useCallback((event: any) => {//or document?
```

Comment:
Rename this callback to onKeyDownListener so that it matches the name of the handled event